main.cpp

```diff
@@ -3,6 +3,17 @@
 #include "ColorPair.h"
 namespace TelCoColorCoder
 {
+    std::string printColorCodingManual(){
+        std::string message = "Color Coding Manual\n";
+        for(int pairNumber = 1; pairNumber <= 25; pairNumber++) {
```

Comment:
Don't hardcode numbers like `25`. They are hard to track and update when changes happen

Reply:
removed the hardcoding